dist/ubuntu/20.04/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ubuntu:20.04 AS build
 
-ENV REFRESHED_AT 20220926T210646Z
+ENV REFRESHED_AT 20221025T182407Z
```

Comment:
Should we install specific version of elixir and erlang? The command below will always grab the latest